lib/onfido/api.rb

```diff
@@ -2,48 +2,52 @@
 
 module Onfido
   class API
-    def initialize(options = {})
-      @api_key = options[:api_key]
+    def initialize(api_key:, region:, **extra_options)
+      @options = Onfido::Options.new(api_key: api_key, region: region, **extra_options)
     end
 
     def applicant
-      Onfido::Applicant.new(@api_key)
+      @applicant ||= Onfido::Applicant.new(options)
```

Comment:
These can now be memoized after getting rid of global config. Should address #19